src/main/java/javax/validation/ConstraintViolationException.java

```diff
@@ -32,7 +34,7 @@ public ConstraintViolationException(String message,
 			this.constraintViolations = null;
 		}
 		else {
-			this.constraintViolations = new HashSet<ConstraintViolation<?>>( constraintViolations );
+			this.constraintViolations = new HashSet<>( constraintViolations );
```

Comment:
I guess it is a matter of taste. For me it feels odd that you can do something like:

```java
constraintViolationException.getConstraintViolations().add(null);
```

However, I'm not strong on this one. I'm fine with leaving it as it is.